includes/admin/functions.php

```diff
@@ -471,7 +471,7 @@ function get_messaging_setting( $field_id, $default = false ) {
  * @param string $field_id The named key in the wp_options serialized array.
  * @return string
  */
-function get_visual_settings( $field_id, $default = false ) {
```

Comment:
Is it safe to change the default?

Reply:
Everywhere that get_visual_settings is called the default is specified as true already. I can't think why we would have made it false at all. Default is the value that the option takes when it has not been set and by default the donation bar should show and the padlock should show next to gated post titles. It felt more intuitive to simply provide the function with its accurate default.

Reply:
@matdehaast @SarahCoilAccount default acts more like fallback here, if there's no option in the database it'll just return false, acting in the same way as WordPress's `get_option`